src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -42,20 +42,39 @@ const ExtensionUpgradeButton = ({ extensionData }: Props) => {
   if (!user || extensionData.isDeprecated || !extensionData.isInitialized) {
     return null;
   }
-  // @TODO check user permissions for canUpgrade - hasRoot(allUserRoles)
+
   const canUpgrade = true;
 
   return (
     <ActionButton
-      appearance={{ theme: 'primary', size: 'medium' }}
       submit={ActionTypes.EXTENSION_UPGRADE}
       error={ActionTypes.EXTENSION_UPGRADE_ERROR}
       success={ActionTypes.EXTENSION_UPGRADE_SUCCESS}
       transform={transform}
-      text={{ id: 'button.upgrade' }}
+      text={{ id: 'button.updateVersion' }}
       disabled={!isSupportedColonyVersion || !extensionCompatible || !canUpgrade}
+      onSuccess={() =>
+        toast.success(
+          <Toast
+            type="success"
+            title="Extension upgraded"
```

Comment:
add translations to tile and description

Reply:
Added